src/main/scala/cromwell/Main.scala

```diff
@@ -42,7 +42,11 @@ object Main extends App {
    * Also now passing args to runAction instead of the constructor, as even sbt seemed to have issues with the args
    * array becoming null in "new Main(args)" when used with: sbt 'run run ...'
    */
-  val AskTimeout = Timeout(5 seconds)
+  // The magic maximum value for an ask timeout.  This value should be large to represent the amount of time until
+  // the future will fail (not the amount of time to wait for the Future to be returned from the ask).  This Future
+  // completes after the workflow goes to a terminal status, so the timeout must be at least as long as the workflow
+  // run duration.
+  val AskTimeout = Timeout(21474835 seconds)
```

Comment:
This needs to bridge the non-actor Main to the Cromwell actor system.  The simplest form of synchronization would seem to be blocking on the completion of a `Promise`, which I believe is essentially what `ask`'s `PromiseActorRef` is [giving us](https://github.com/akka/akka/blob/a7021827c3974762143920139a72f33064dd78aa/akka-actor/src/main/scala/akka/pattern/AskSupport.scala#L597).

Granted this seems like a weird usage, but it is a weird use case.  :smile: